lib/IDE/CodeCompletion.cpp

```diff
@@ -2869,10 +2878,12 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
       Builder.setAssociatedDecl(SD);
       setClangDeclKeywords(SD, Pairs, Builder);
     }
-    if (!HaveLParen)
+    if (!HaveLParen) {
       Builder.addLeftBracket();
-    else
+    } else {
+      Builder.addFlair(CodeCompletionFlairBit::ArgumentLabels);
```

Comment:
Why does the `HaveLParen` branch not have the flair?  Looking at the test cases I see for a `init#^A^#`:
```
Decl[Constructor]/CurrNominal/Flair[ArgLabels]: ()[#SuperBaseA#]; name=()
```
We should probably be consistent about whether `(...)` is considered `ArgumentLabels` or not.

Reply:
This is `addSubscriptCallPattern`.
For `expr#^HERE#^#`, suggest `[index: <#Index#>]` along with other methods.
For `expr[#^HERE^#`], suggest `index: <#Index#>` as an "argument labels"

Reply:
Why is it different for subscript vs. other calls though?  Like I said, for `init#^HERE^#` it *is* getting `ArgumentLabels`.

Reply:
For `expr #^HERE^#`, I consider suggesting `[index: <#Index#>]` is suggesting `subscript` itself, not call arguments for the subscript, because the user has not input anything for the subscript.

As for `init#^HERE^#`, the user has input `.init`, so "call arguments" is relevant *I think*.

Reply:
I will add a comment explaining this.